ParisBNF/abb/BNFabb47.xml

```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb47" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Four Gospels</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+            <date>2016-06-07T17:45:12.922+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno>BnF Éthiopien d'Abbadie 47</idno>
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
+
+                     <msItem xml:id="ms_i1">
+                       <locus from="1r"/>
+                        <title type="complete" ref="LIT1560Gospel"/>
+
+
+                        <msItem xml:id="ms_i1.1">
+                          <locus from="1r"/>
+                           <title type="incomplete" ref="LIT2705Introd">Prefatory part</title>
+
+                           <msItem xml:id="ms_i1.1.1">
+                             <locus from="1r" to="8v" cert="medium"/>
```

Comment:
I know that writing something like cert="medium" can seem satisfactory to us, but I don't think that it's actually very meaningful, because (at least to me) it is still unclear what is how uncertain. I also don't think that it is printed on the app, so to most readers it will stay hidden. Maybe in addition, an explicit sentence in a `<note>` stating what is uncertain could be helpful.

Reply:
done, enough like this? the catalogues leave some space for speculation, as to what the ms includes.